### PR TITLE
Upgrade to @fastly/js-compute@1.3.2

### PR DESCRIPTION
**Pull request by doramatadora**
- Upgrades to @fastly/js-compute@1.3.2
- Specifies node LTS

### CODE REVIEW

**Comment by harmony7** [comment]
I'm good with removing Webpack from this, but if we're going to do that, I think we should make (fork?) a starter kit that includes Webpack. Webpack is still necessary for compute@edge in many cases because it's needed for shimming and polyfilling, often needed for bringing in modules from npm.

File: fastly.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-manifest_version = 2
-name = "Default starter for JavaScript"
-description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
+description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 language = "javascript"
+manifest_version = 2
+name = "Default starter for JavaScript"
```
Comment (harmony7) [inline fastly.toml]:
```suggestion
manifest_version = 2
name = "Default starter for JavaScript"
description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
authors = ["<oss@fastly.com>"]
language = "javascript"
```

These changes are unnecessary.

**Review by harmony7 — APPROVED** [review]
Looks good to me, with two points:

* `fastly.toml` is changed for no good reason
* please make a separate version of this starter kit that still uses Webpack as that's useful (and necessary for many npm packages, frameworks, etc)

**Comment by doramatadora** [comment]
> * please make a separate version of this starter kit that still uses Webpack as that's useful (and necessary for many npm packages, frameworks, etc)

Having reconsidered, I think it's best for a generic starter like this to include webpack support (e.g. to give people a pattern for working with asset modules) so I have reinstated webpack. The changeset is just upgrading dependencies now.

**Pull request merged**